cmd/scaffold.go

```diff
@@ -322,7 +324,8 @@ func (fnb *FlowNodeBuilder) initMetrics() {
 
 	fnb.Tracer = trace.NewNoopTracer()
 	if fnb.BaseConfig.tracerEnabled {
-		tracer, err := trace.NewTracer(fnb.Logger, fnb.BaseConfig.NodeRole)
+		serviceName := fnb.BaseConfig.NodeRole + "-" + fnb.BaseConfig.nodeIDHex[:8]
```

Comment:
I wonder if we could maybe add network name to avoid confusion, I could imagine we can have multiple nodes (on sporks or networks) running at the same time

Reply:
I think the node Id and blockIDs are unique per networks, not sure we would hit that problem.

Reply:
I did refactor so the root span now includes a chainID tag, so we can filter out traces.